LambdaRuntimeDockerfiles/Infrastructure/README.md

```diff
@@ -12,97 +12,34 @@ Infrastructure project allows to create pipeline to build and push .NET 5 Lambda
--SourceCrossAccountRoleArn "arn:aws:iam::SourceAccountId:role/aws-lambda-dotnet-source-account-role" `
--SourceBranchName "main" `
--StageEcr "AccountId.dkr.ecr.us-west-2.amazonaws.com" `
--BetaEcrs "AccountId.dkr.ecr.us-west-2.amazonaws.com;AccountId.dkr.ecr.us-west-2.amazonaws.com" `
--ProdEcrs "AccountId.dkr.ecr.us-west-2.amazonaws.com;AccountId.dkr.ecr.us-west-2.amazonaws.com" `
--EcrRepositoryName "awslambda/dotnet5.0-runtime;awslambda/dotnet6.0-runtime" `
--Profile "default" `
 -TargetFramework "net5;net6" `
 -DotnetChannel "5.0;6.0"
 ```
 
-`arn:aws:iam::SourceAccountId:role/aws-lambda-dotnet-source-account-role` must have following permissions and trust relationships in `SourceAccountId`.
-
-**Permissions**
-```json
-{
-    "Version": "2012-10-17",
-    "Statement": [
-        {
-            "Effect": "Allow",
-            "Action": [
-                "codecommit:UploadArchive",
-                "kms:Decrypt",
-                "s3:PutObject*",
-                "codecommit:CancelUploadArchive",
-                "s3:GetObject*",
-                "kms:ReEncrypt*",
-                "kms:GenerateDataKey*",
-                "codecommit:GetCommit",
-                "codecommit:GetUploadArchiveStatus",
-                "s3:Abort*",
-                "s3:List*",
-                "kms:Encrypt",
-                "codecommit:GetBranch",
-                "kms:DescribeKey",
-                "s3:GetBucket*",
-                "s3:DeleteObject*"
-            ],
-            "Resource": [
-                "arn:aws:kms:*:CdkAccountId:key/*",
-                "arn:aws:codecommit:us-west-2:SourceAccountId:aws-lambda-dotnet",
-                "arn:aws:s3:::aws-lambda-dotnet-5-cont-pipeline*"
-            ]
-        }
-    ]
-}
-```
-
-**Trust Relationships**
-```json
-{
-  "Version": "2012-10-17",
-  "Statement": [
-    {
-      "Effect": "Allow",
-      "Principal": {
-        "AWS": "arn:aws:iam::CdkAccountId:root" 
-      },
-      "Action": "sts:AssumeRole",
-      "Condition": {}
-    }
-  ]
-}
-```
-
-For consistency purpose, same ECR repository name is used across all ECRs.
+#### Notes
+ - AWS Profiles used to execute `bootstrap.ps1` must have administrator access.
+ - All resources used to bootstrap the pipeline must already exist.
+ - `AccountId` is AWS AccountId used for deploying CDK App.
+ - `CodeCommitAccountId` is AWS AccountId that contains source repository.
+ - If the CodeCommit repository is in the same account, use the same account Id for `PipelineAccountId` and `CodeCommitAccountId`.
+ - When doing a cross-account deployment, you need to have AWS Profiles for both accounts.
+ - `bootstrap.ps1` will run 2 `cdk bootstrap` commands for the cross account deployments to establish a trust relationships between the accounts. This way, we do not require a separate IAM role to be created manually.
+ - For consistency purpose, same ECR repository name is used across all ECRs.
```

Comment:
This note doesn't make a lot of sense. I assume it means the same repository name is used no matter what AWS account is being used but it is worded weird.

Reply:
Are you talking about the whole block? or the last bullet point?

Reply:
Just the last bullet point